src/main/java/org/humancellatlas/ingest/project/ProjectRepositoryImpl.java

```diff
@@ -14,14 +13,15 @@
 import java.util.ArrayList;
 import java.util.Collection;
 import java.util.List;
+import java.util.Optional;
 
 public class ProjectRepositoryImpl implements ProjectRepositoryCustom {
 
     @Autowired
     private MongoTemplate mongoTemplate;
 
     @Override
-    public Page<Project> findByContent(List<MetadataCriteria> metadataQuery, Pageable pageable) {
+    public Page<Project> findByContent(List<MetadataCriteria> metadataQuery, Optional<Boolean> isUpdate, Pageable pageable) {
```

Comment:
should we just allow users to query by document fields and not just content fields?

Reply:
That is a better long term solution. Will look into it.